src/open_inwoner/scss/components/Form/NonFieldError.scss

```diff
@@ -0,0 +1,6 @@
+.form {
+  &__non-field-error {
```

Comment:
This is incorrect, if a non field error is a component (or) block on it's own it should not be an element of something else (form). This is mixture happens on other places as well IIRC (form__control?) but lets not increase the usage of that pattern here.

The correct block name IMO should be "non-field-error" and should not contain the form class name.